src/dmd/cond.d

```diff
@@ -658,6 +658,7 @@ extern (C++) final class VersionCondition : DVCondition
             case "CppRuntime_Gcc":
             case "CppRuntime_Microsoft":
             case "CppRuntime_Sun":
+            case "Cpp98":
```

Comment:
Does this need to be added to the predefined version tests?

Reply:
yes